.github/workflows/close-inactive-issues.yml

```diff
@@ -0,0 +1,60 @@
+name: Close inactive issues
+on:
+  schedule:
+    - cron: "30 1 * * *" # once a day (1:30 UTC)
+
+jobs:
+  close-issues-without-assignee:
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+      pull-requests: write
+    steps:
+      - uses: actions/stale@v5
+        with:
+          days-before-issue-stale: 7
```

Comment:
I would at least give it 14 days until stale, and another 7 until close. 7+3 is a pretty short time frame, considering that there are holiday periods etc.